acdc/TLACDCEdge.py

```diff
@@ -29,7 +31,6 @@ class EdgeType(Enum):
     PLACEHOLDER = 2
 
     def __eq__(self, other):
-        # TODO WTF? Why do I need this?? To busy to look into now, check the commit where we add this later
```

Comment:
Can you explain why?

Reply:
I got some weird old bug without this. It's probably safer to delete this anyway, and the old bug did not reproduce